ledger-core/v2/insolar/pulse/memory_test.go

```diff
@@ -24,11 +25,11 @@ func TestNodeStorage_ForPulseNumber(t *testing.T) {
 	storage := NewStorageMem()
 	storage.storage[pn] = &memNode{pulse: pulse}
 
-	t.Run("returns error when no Pulse", func(t *testing.T) {
-		res, err := storage.ForPulseNumber(ctx, gen.PulseNumber())
-		assert.Equal(t, ErrNotFound, err)
-		assert.Equal(t, insolar.Pulse{}, res)
-	})
+	// t.Run("returns error when no Pulse", func(t *testing.T) {
+	// 	res, err := storage.ForPulseNumber(ctx, gen.PulseNumber())
+	// 	assert.Equal(t, ErrNotFound, err)
+	// 	assert.Equal(t, insolar.Pulse{}, res)
+	// })
```

Comment:
FYI generally we don't keep commented code. It can always be recovered using VCS if necessary. I believe it somewhere in our code style.

Reply:
Well, I like to revisit code multiple times, and comments are more efficient. VCS is only good when you know where and which version to look at. It is very inefficient otherwise.

I've restored this test